packages/recs/src/Query.ts

```diff
@@ -508,7 +526,7 @@ export function defineQuery(
 
   return {
     matching,
```

Comment:
from an API standpoint, I wonder if we should return `matching.value` instead of the observable (esp. since the BehaviorSubject is mutable, which is kinda scary)

then we don't have to change all the other locations to use `matching.value`